### PR TITLE
https://github.com/vmware/terraform-provider-vra/issues/132

### PR DESCRIPTION
**Pull request by Prativa20**
Support for resize block device vra on prem and cloud

Signed-off-by: Prativa Bawri <bawrip@vmware.com>

### CODE REVIEW

File: vra/resource_block_device.go [hunk 1]
```diff
@@ -260,120 +271,43 @@ func resourceBlockDeviceUpdate(d *schema.ResourceData, m interface{}) error {
...
-			}
-			log.Printf("Finished resize of vra_block_device resource with name %s", d.Get("name"))
-		} else {
-			return fmt.Errorf("Resize action is not available for the resource with name %s", d.Get("name"))
-		}
-	}
+	blockDeviceIDs := resourceIDs.([]string)
+	i := strings.LastIndex(blockDeviceIDs[0], "/")
+	blockDeviceID := blockDeviceIDs[0][i+1 : len(blockDeviceIDs[0])]
+	d.SetId(blockDeviceID)
```
Comment (dmettem) [inline vra/resource_block_device.go]: Does the id change after resize?

File: website/docs/d/block_device.html.markdown [hunk 2]
```diff
@@ -108,6 +108,8 @@ The following arguments are supported for a machine resource:
 
 * `project_id` - The id of the project the current user belongs to.
 
+*`persistent` - Indicates whether the block device survives a delete action.
```
Comment (dmettem) [inline website/docs/d/block_device.html.markdown]: Nit. Space after the `*`.

File: website/docs/r/block_device.html.markdown [hunk 3]
```diff
@@ -130,6 +131,8 @@ Following disk custom properties can be passed while creating a block device:
 
 * `encrypted` - (Optional) Indicates whether the block device should be encrypted or not.
 
+*`persistent` - Indicates whether the block device survives a delete action.
```
Comment (dmettem) [inline website/docs/r/block_device.html.markdown]: Nit. Space after the `*`.

File: vra/resource_block_device.go (hunk 1 above)
Comment (Prativa20) [reply vra/resource_block_device.go]: No, removed this logic

File: website/docs/d/block_device.html.markdown (hunk 2 above)
Comment (Prativa20) [reply website/docs/d/block_device.html.markdown]: done

File: website/docs/r/block_device.html.markdown (hunk 3 above)
Comment (Prativa20) [reply website/docs/r/block_device.html.markdown]: done

**Pull request merged**